src/tasks/AddTask.tsx

```diff
@@ -40,13 +41,30 @@ export const AddTask = ({
     display?: 'chip' | 'icon';
 }) => {
     const { identity } = useGetIdentity();
+    const dataProvider = useDataProvider();
+    const notify = useNotify();
     const { taskTypes } = useConfigurationContext();
     const contact = useRecordContext();
     const [open, setOpen] = useState(false);
     const handleOpen = () => {
         setOpen(true);
     };
 
+    const handleSuccess = async (data: any) => {
+        setOpen(false);
+        const contact = await dataProvider.getOne('contacts', {
+            id: data.contact_id,
+        });
+        if (!contact.data) return;
+
+        dataProvider.update('contacts', {
+            id: contact.data.id,
+            data: { last_seen: new Date().toISOString() },
+            previousData: contact.data,
+        });
```

Comment:
**Suggestions:**

- You should `await` the mutaiton before displaying the success notification
- You should use `useUpdate` from `react-admin` instead of calling the dataProvider directly, as this will allow to update the react-query cache, so that the data is up to date without having to wait for the refresh on the ContactList page